main.tex

```diff
@@ -938,6 +938,48 @@ \subsection{Path}\label{sec:path-benchmark}
 Taken together, our benchmarks show that our method performs better
 than all the competing methods.
 
+\section{Application to real-world data}\label{sec:realdat}
```

Comment:
I think we can make this more specific, like "Application to metabolomic data" or something similar.